.github/workflows/release_isolate_proto.yml

```diff
@@ -0,0 +1,71 @@
+name: PyPI Release isolate-proto
```

Comment:
this releases isolate-proto AND fal because of 

```
  push:
    branches:
      # React to pushes to 'bump-proto-version-*' branch to release 'fal'
      # whenever we have a new 'isolate-proto' version
      - bump-proto-version-*
```

in release.yml

Let's change the name or remove that workflow event